crates/swc/src/config/mod.rs

```diff
@@ -486,7 +486,8 @@ impl Options {
                 env_name: self.env_name.to_owned(),
             };
 
-            crate::plugin::plugins(experimental, plugin_context)
+            swc_plugin_runner::cache::init_plugin_module_cache_once();
```

Comment:
I think we should check if `plugins` is not empty

Reply:
Otherwise, we create a directory named `.swc` even though the user does not use plugin.

Reply:
this is for the wasm target, where `init_plugin_module_cache_once` does not have access to filesystem and its implementation actually do not create any file - only creates hashmap for in memory cache.

Reply:
Oh, I see. Thanks!